pkg/api/controller_test.go

```diff
@@ -176,6 +186,25 @@ func TestController_GetRepoHandler(t *testing.T) {
 	})
 }
 
+func testCommitEntries(t *testing.T, ctx context.Context, cat catalog.Interface, deps *dependencies, params commitEntriesParams) string {
+	t.Helper()
+	s := strconv.Itoa(params.seed)
+	for _, p := range params.paths {
+		err := cat.CreateEntry(ctx, params.repo, params.branch,
+			catalog.DBEntry{
+				Path:            p,
+				PhysicalAddress: onBlock(deps, "add"+s+"of:"+p),
+				CreationDate:    time.Now(),
+				Size:            int64(params.seed),
+				Checksum:        "cksum" + s,
```

Comment:
I don't understand `s` here: it will always be the same in the loop, so all `Checksum`s are equal.  If we don't care then just keep a constant `Checksum`, but if we do then please increment it or seed a PRNG or something.

Reply:
I'm passing seed and choosing it according to commit graph I created. I'm using different seed when I want to create a commit in which a file changed. Sounds Ok?

Reply:
Maybe the word "seed" in confusing me.  I understand it in this computing context, of a [PRNG seed](https://en.wikipedia.org/wiki/Random_seed).  Using the value and never growing anything from it confuses me, that's not what a seed does.

Also I don't understand why use `s` here and on line 196, but `params.seed` on l.198.  Also the type of `params.seed` is strange: it is literally cast *everywhere* it is used.

Can we just rename it to reflect what it is, give it type `int64`, and then, rather than `strconv` it and concatenate, use `fmt.Sprintf`?